app/views/citizens/transactions/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= page_template page_title: t("transaction_types.page_titles.#{@transaction_type.name}"), template: :default do %>
+
+  <h2 class="govuk-heading-m"><%= t '.sub_heading' %></h2>
+
+  <div class="govuk-!-padding-bottom-4"></div>
+
+  <%= form_with(url: citizens_transactions_path, method: :patch, local: true) do |form| %>
+
+    <%= form.hidden_field :transaction_type, value: @transaction_type.name %>
+
+    <table class="govuk-table">
+      <caption class="govuk-table__caption"><%= t('date.date_period', from: date_from, to: date_to) if @bank_transactions.any? %></caption>
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col"><%= t '.col_select_all' %></th>
+          <th class="govuk-table__header" scope="col"><%= t '.col_date' %></th>
+          <th class="govuk-table__header" scope="col"><%= t '.col_description' %></th>
+          <th class="govuk-table__header" scope="col"><%= t '.col_amount' %></th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <%= form.collection_check_boxes :transaction_ids, @bank_transactions, :id, :description do |builder| %>
+          <tr class="govuk-table__row">
+            <th class="govuk-table__cell" scope="row">
+              <div class="govuk-checkboxes__item">
+                <% checked = builder.object.transaction_type_id == @transaction_type.id %>
+                <%= builder.check_box(class: 'govuk-checkboxes__input', checked: checked) %>
+                <label class="govuk-label govuk-checkboxes__label"></label>
```

Comment:
I think an empty label tag here may cause accessibility issues.

I think the "label" is the description and/or happened at date rather than the input field. So the `scope="row" belongs on these cells rather than the input cell.

Reply:
As per our conversation offline, if the label is not present, the checkbox vanishes. 🤷‍♂️